types/iobroker/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for ioBroker 2.0
+// Type definitions for ioBroker 2.2
```

Comment:
The major and minor version of these type definitions should match that of the package they reflect, so I recommend leaving this as 2.0 (unless this is a pre-emptive change for a forthcoming release of iobroker v2.2).

Reply:
@uniqueiniquity This is indeed for the 2.2 release.

By the way, why is your nickname so hard to type? 😅

Reply:
I thought it would be fun when I came up with it years ago... clearly wasn't thinking ahead enough. 😆 Might be worth a change someday.

Reply:
I thought it would be fun when I came up with it years ago... clearly wasn't thinking ahead enough. 😆 Might be worth a change someday.